File: reservations - API/reservations/ProjectCalculator.Domain/Domain/Address.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Reservations.Core.Domain
{
    public class Address
    {
        public Guid Id { get; set; }
        public string Street { get;  set; }
        public string City { get;  set; }
        public string ZipCode { get;  set; }
    }
}

File: reservations - SPA/reservations-spa/src/app/_models/Address.ts
export class Address{
    public id:number;
    public street:string;
    public city:string;
    public zipCode:string;
    
    constructor(id:number, street:string, city:string, zipCode:string){
        this.id = id;
        this.street = street;
        this.city = city;
        this.zipCode = zipCode;
    }
}